main.go

```diff
@@ -108,227 +101,208 @@ func runTests(machine *client.RobotClient) {
 	}
 
 	// wheeled base tests
-	logger.Info("Starting wheeled base tests...")
```

Comment:
we should still keep these logs, so that if we wanted to parse the logfile we can still see all of the tests running

Reply:
sure I'll readd them